doc/whats-new.rst

```diff
@@ -12,6 +12,10 @@ What's New
 v0.6.0 (unreleased)
 -------------------
 
+- Added new methods :py:meth:`~xray.DataArray.diff` and
+  :py:meth:`~xray.Dataset.diff` for finite difference calculations
```

Comment:
This ends up looking  like `Added new methods diff() and diff()` once you build the docs. It would be better to only include one or them or use:

    :py:meth:`DataArray.diff <xray.DataArray.diff>`